components/prism-diff.js

```diff
@@ -39,11 +39,24 @@
 			alias.push("bold");
 		}
 
-		Prism.languages.diff[name] = {
-			// pattern: /^(?:[_].*(?:\r\n?|\n|(?![\s\S])))+/m
-			pattern: RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm'),
-			alias: alias
-		};
+		var pattern = RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm');
+
+		if (['-', '<', '+', '>'].indexOf(prefix) !== -1) {
```

Comment:
I don't see why you want to add the prefix token for a select few. 

The motivation is that you don't want to copy the prefix, right? Then why is it ok to copy e.g. the space prefix?

Reply:
Because whitespace won't stop the code from running but `+` `-` will, which means we have to remove`+` `-` etc. to make the code work. Also, Code Formatter like prettier could fix the whitespace problem easily.

Reply:
I don't know what language you're using but python code will definitely stop running with an extra space.

Also, what about the `!` prefix?

Reply:
I use HTML/CSS/JavaScript mostly. I would rather cover use cases I already known. When people using other languages want to target those prefixes too, they could improve it later. Handling something I'm not familiar with is pretty dangerous.

Reply:
> they could improve it later.

Most people will probably just use another syntax highlighter instead of submitting a PR. (Thank you for doing the latter)

> Handling something I'm not familiar with is pretty dangerous.

Agreed but in this case, it's pretty safe. 
Worst-case: The highlighting is a bit off.
Worst-worst-case: Some extra characters get copied.

From my point of view, handling all prefixes equally is a better solution. It's simpler, works for all, and is [less surprising](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) (why only these prefixes?).

Reply:
I agree with you.